service/src/test/java/org/eclipse/keti/acs/service/policy/evaluation/ConditionEvaluationTest.java

```diff
@@ -24,196 +24,161 @@
 import java.util.List;
 import java.util.Set;
 
+import org.apache.commons.lang.StringUtils;
 import org.eclipse.keti.acs.commons.policy.condition.groovy.GroovyConditionShell;
-import org.eclipse.keti.acs.commons.policy.condition.groovy.NonCachingGroovyConditionCache;
 import org.eclipse.keti.acs.model.Attribute;
 import org.eclipse.keti.acs.model.Condition;
-import org.eclipse.keti.acs.rest.BaseResource;
-import org.eclipse.keti.acs.rest.BaseSubject;
-import org.eclipse.keti.acs.service.policy.validation.PolicySetValidator;
-import org.eclipse.keti.acs.service.policy.validation.PolicySetValidatorImpl;
 import org.mockito.internal.util.reflection.Whitebox;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
 import org.testng.Assert;
-import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 /**
  *
  * @author acs-engineers@ge.com
  */
-@ContextConfiguration(
-        classes = { NonCachingGroovyConditionCache.class, GroovyConditionShell.class, PolicySetValidatorImpl.class })
+@ContextConfiguration(classes = { GroovyConditionShell.class })
 public class ConditionEvaluationTest extends AbstractTestNGSpringContextTests {
 
+    private static final String ATTRIBUTE_ISSUER = "acs";
+    private static final Attribute SITE_SAN_RAMON = getAttribute("site", "San Ramon");
+    private static final Attribute SITE_NEW_YORK = getAttribute("site", "New York");
+    private static final Attribute SITE_BOSTON = getAttribute("site", "Boston");
+    private static final Attribute SITE_LA = getAttribute("site", "LA");
+    private static final Attribute LOCATION_SAN_RAMON = getAttribute("location", "San Ramon");
+    private static final Attribute LOCATION_NEW_YORK = getAttribute("location", "New York");
+    private static final Attribute DEPARTMENT_SALES = getAttribute("department", "sales");
+
+    private PolicyEvaluationServiceImpl evaluationService;
+
     @Autowired
-    private PolicySetValidator policySetValidator;
+    private GroovyConditionShell conditionShell;
 
-    @BeforeClass
-    public void setup() {
-        ((PolicySetValidatorImpl) this.policySetValidator)
-                .setValidAcsPolicyHttpActions("GET, POST, PUT, DELETE, PATCH, SUBSCRIBE, MESSAGE");
-        ((PolicySetValidatorImpl) this.policySetValidator).init();
+    @BeforeMethod
+    private void setupMethod() throws Exception {
+        this.evaluationService = new PolicyEvaluationServiceImpl();
+        Whitebox.setInternalState(this.evaluationService, "conditionShell", this.conditionShell);
```

Comment:
We should stop using `Whitebox` altogether since it's in the `org.mockito.internal` package which is subject to breaking changes. Prefer Spring's test utilities instead:
```suggestion
        ReflectionTestUtils.setField(this.evaluationService, "conditionShell", this.conditionShell);
```